odoo/tests/common.py

```diff
@@ -419,6 +419,16 @@ def run(self, result):
             if not failure:
                 break
 
+    @classmethod
+    def setUpClass(cls):
+        def check_remaining_patchers():
+            for patcher in _patch._active_patches:
+                _logger.error("A patcher was remaining active at the end of %s", cls.__name__)
```

Comment:
```suggestion
                _logger.warning("A patcher was remaining active at the end of %s, disabling it...", cls.__name__)
```

error is unrecoverable issue that the user must take action on
warning is silly thing that shouldn't happen, might reveal a real bug underlying but the system was able to recover by itself